packages/flutter/lib/src/services/text_input.dart

```diff
@@ -825,6 +828,10 @@ class TextInput {
 
   static bool _debugEnsureInputActionWorksOnPlatform(TextInputAction inputAction) {
     assert(() {
+      if (_kIsBrowser) {
+        // TODO(flutterweb): what makes sense here?
```

Comment:
nit: let's spell it `TODO(flutter_web)` so our regexes can find this TODO later.